src/routes/chat/ChatOffer/ChatOffer.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import './ChatOffer.scss';
+
+const ChatOffer = () => {
+  return (
+    <div>
+      <div className="container">
+        <h1 className="is-size-5 chatoffer__header">YOUR BID</h1>
+        <h1 className="is-size-5 chatoffer__bids">
+          <div className="columns is-mobile">
+            <div className="column chatoffer__bids--text">Bid #0</div>
+            <div className="column chatoffer__bids--text">Quantity: 300</div>
+            <div className="column chatoffer__bids--text">Price: SGD 6.89</div>
+          </div>
+        </h1>
+        <div className="columns is-gapless is-mobile">
+          <div className="column">
+            <button
```

Comment:
Add issue to this repo and add TODO so we don't forget.
`//TODO(#issueNumber): add make offer button in chat functional`